custom_components/google_home/const.py

```diff
@@ -89,8 +90,128 @@
 # JSON parameter values when retrieving information from devices
 JSON_ALARM: Final[str] = "alarm"
 JSON_TIMER: Final[str] = "timer"
-JSON_NOTIFICATIONS_ENABLED: Final[str] = "notifications_enabled"
 JSON_ALARM_VOLUME: Final[str] = "volume"
+JSON_NOTIFICATIONS_ENABLED: Final[str] = "notifications_enabled"
+JSON_EUREKA_DEVICE_INFO: Final[str] = "device_info"
+JSON_EUREKA_CAPABILITIES: Final[str] = "capabilities"
+
+# EUREKA
+EUREKA_KEY_PARAMS = "params"
```

Comment:
Actually just `Final` should be enough, you may strip all `[str]` here.
```suggestion
EUREKA_KEY_PARAMS: Final[str] = "params"
```